prices_voice.go

```diff
@@ -48,26 +58,32 @@ type VoicePrice struct {
 }
 
 type VoiceNumberPrice struct {
-	Country           string            `json:"country"`
-	IsoCountry        string            `json:"iso_country"`
-	Number            string            `json:"number"`
-	InboundCallPrice  InboundPrice      `json:"inbound_call_price"`
-	OutboundCallPrice OutboundCallPrice `json:"outbound_call_price"`
-	PriceUnit         string            `json:"price_unit"`
-	URL               string            `json:"url"`
+	Country            string              `json:"country"`
+	IsoCountry         string              `json:"iso_country"`
+	DestinationNumber  string              `json:"destination_number"`
+	OriginationNumber  string              `json:"origination_number"`
+	InboundCallPrice   InboundPrice        `json:"inbound_call_price"`
+	OutboundCallPrices []OutboundCallPrice `json:"outbound_call_prices"`
+	PriceUnit          string              `json:"price_unit"`
+
+	URL string `json:"url"`
 }
 
+// https://www.twilio.com/docs/voice/pricing#pricing-voice-country-instance-resource
 // returns the call price by country
 func (cvps *CountryVoicePriceService) Get(ctx context.Context, isoCountry string) (*VoicePrice, error) {
 	voicePrice := new(VoicePrice)
 	err := cvps.client.GetResource(ctx, voicePathPart+"/Countries", isoCountry, voicePrice)
 	return voicePrice, err
 }
 
-// returns the call price by number
-func (nvps *NumberVoicePriceService) Get(ctx context.Context, number string) (*VoiceNumberPrice, error) {
+// https://www.twilio.com/docs/voice/pricing#list-uri-by-origination-number
+// returns the call price by number or numbers pair
+func (nvps *NumberVoicePriceService) Get(ctx context.Context, destinationNumber string,
```

Comment:
I don't think this is quite right. The URL structure is:

```
https://pricing.twilio.com/v2/Voice/Numbers/{Destination Number}?OriginationNumber={Origination Number}
```

Elsewhere in the API, we handle parameters that are part of the path by adding them as required fields as we've done here, but we handle query parameters as a `url.Values`. So I think the API should be:

```
Get(ctx context.Context, destinationNumber string, data url.Values)
```

We can indicate either in the docstring or an example (or both) that the one valid value for `data` currently is `OriginationNumber`. But I think it is a mistake to include it as part of the method declaration - what if Twilio adds another parameter or changes its type? The url.Values method is future proof, hand coding arguments is not.

Reply:
As a side effect this means you can get rid of `numberVoicePricePart` I think and just pass through a url.Values

Reply:
done